project_task_description_template/models/task_template.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+from odoo import fields, models
+
+
+class TaskTemplate(models.Model):
+    _name = "project.task.template"
+    _description = "Project Task Template"
+
+    name = fields.Char()
+    tag_ids = fields.Many2many(comodel_name="project.tags")
+    user_id = fields.Many2one(string="Assigned to", comodel_name="res.users")
+    description = fields.Html()
```

Comment:
Why do you use Html field?

Reply:
because in project.task 
https://github.com/odoo/odoo/blob/4c8d544c8c50bbc1ff646edcc4b130c0ccf0ef91/addons/project/models/project.py#L571